dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/profiler/GlobalBenchmark.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.api.profiler
+
+import com.datadog.android.lint.InternalApi
+
+/**
+ * A global holder of [BenchmarkProfiler], allowing to register and retrieve [BenchmarkProfiler] implementation.
+ * This should only used by internal benchmarking.
+ */
+@InternalApi
+object GlobalBenchmark {
+
+    private var benchmarkProfiler: BenchmarkProfiler = NoOpBenchmarkProfiler()
+
+    /**
+     * Registers the implementation of [BenchmarkProfiler].
+     */
+    @InternalApi
```

Comment:
InternalApi is both on the singleton and on the method. Doesn't the annotation at the class level preclude adding at the method level?

Reply:
You are right, removed.